test/utils/asset.spec.js

```diff
@@ -391,6 +391,49 @@ describe('services', () => {
       it('should return an empty array if the asset URI contains root URI but not as a prefix', () => {
         expect(AssetUtil.findAllDescendantAssetsByUri('/Test/Rt/Child', '/Rt')).toBeArrayOfSize(0);
       });
+      it.onWindows('should return an empty array if the asset URI is not specified', () => {
```

Comment:
Thank you for adding the Windows tests.  Unfortunately these are failing for me on my machine.  Did all tests pass for you?

Reply:
I actually tested them on unix, which now I think have been the best choice for validating Windows-specific tests. I assumed that by using the `.onWindows` or `.onMac` annotations, the tests would execute independently of the underlying system. 
But, am still scratching my head for how did they all pass, given that I passed the windows based filepaths in testcases.
![Screenshot from 2024-04-03 20-34-37](https://github.com/StatTag/StatWrap/assets/96688318/83cffb41-c901-48e2-b964-1d228792626e)

Reply:
My bad. I assumed that by using the `.onWindows` or `.onMac` annotations, the tests would execute independently of the underlying system. I tested on Unix

Reply:
No worries!